cmd/bnscli/common.go

```diff
@@ -158,3 +160,36 @@ const txHeaderSize = 4
 type stater interface {
 	Stat() (os.FileInfo, error)
 }
+
+// readAddressWeightPairCSV reads all address weight pairs from a csv file and calls the callback.
+func readAddressWeightPairCSV(csvpath string, f func(address weave.Address, weight uint32)) error {
```

Comment:
It might be better to keep this in the `cmd_distribution.go` file as this is the only place that is using it now. 
Reading CSV should be replaced there with updating the transaction instead. This is a task for another pull request though.

Reply:
I am not sure if we can get it replaced as this was a requirement to support csv. In fact I think we may have to add it back here fore convenience reasons. 🤷‍♂ 
Will revert the extract method refactoring meanwhile